### PR TITLE
Add notification settings page & runtime notifications

### PR DESCRIPTION
**Pull request by SuZhou-Joe**
### Description
[Describe what this change achieves]

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by xluo-aws — APPROVED** [review]

**Review by xluo-aws — DISMISSED** [review]

File: .github/workflows/cypress-workflow.yml [hunk 1]
```diff
@@ -35,6 +35,9 @@ jobs:
           cd index-management
           ./gradlew run -Dopensearch.version=${{ env.OPENSEARCH_VERSION }} &
           sleep 300
+        # cd index-management
+        # ./gradlew run -PcustomDistributionUrl="https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.7.0/7747/linux/x64/tar/dist/opensearch/opensearch-2.7.0-linux-x64.tar.gz" &
```
Comment (xluo-aws) [inline .github/workflows/cypress-workflow.yml:39]: Should those new comment out lines be removed?

Comment (SuZhou-Joe) [reply .github/workflows/cypress-workflow.yml:39]: Sure, and this PR should be merged after the notification PR.

File: public/containers/NotificationConfig/NotificationCallout.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  permissionForCreateLRON: boolean;
+}
+
+export default function NotificationCallout(props: INotificationCalloutProps) {
+  const { hasDefaultNotification, operationType, actionType, permissionForCreateLRON, permissionForViewLRON } = props;
+  const permissionValue = getPermissionValue(permissionForViewLRON, permissionForCreateLRON, hasDefaultNotification);
+
+  if (permissionValue === "110") {
+    return null;
+  }
```
Comment (Hailong-am) [inline public/containers/NotificationConfig/NotificationCallout.tsx:21]: what does 110 means? better to have enum to define all values.

Comment (SuZhou-Joe) [reply public/containers/NotificationConfig/NotificationCallout.tsx:21]: Already add comment on permissionValue.

**Review by Hailong-am — APPROVED** [review]

**Review by xluo-aws — APPROVED** [review]

**Pull request merged**